app/code/Magento/Catalog/Model/Indexer/Category/Product/Action/Full.php

```diff
@@ -174,7 +174,7 @@ protected function reindex(): void
         foreach ($this->storeManager->getStores() as $store) {
             if ($this->getPathFromCategoryId($store->getRootCategoryId())) {
                 $userFunctions[$store->getId()] = function () use ($store) {
-                    return $this->reindexStore($store);
+                    $this->reindexStore($store);
```

Comment:
is this change related to this pr ?

Reply:
This suggestion by PHPStan. "reindexStore" return type is defined as void, so there needs to return result